draft-ietf-dots-signal-channel-26.xml

```diff
@@ -1382,7 +1385,7 @@
         "target-protocol": [
           6
         ],
-        "lifetime": 1800,
+        "lifetime": 1755,
```

Comment:
I wanted to make these lifetimes both not round numbers (since they are the time remaining, they will generally not be round numbers) and different values, to indicate that the mitigation requests may have been placed in close succession but not at exactly the same time.